src/timetables/day_to_timetable.rs

```diff
@@ -171,6 +171,28 @@ where
         has_day_to_timetable.get(&local_zone)
     }
 
+    pub fn get_base_timetable(
+        &self,
+        vehicle_journey_idx: &VehicleJourneyIdx,
+        local_zone: LocalZone,
+        day: DaysSinceDatasetStart,
+        days_patterns: &DaysPatterns,
+    ) -> Option<Timetable> {
+        let day_to_timetable = self.get_day_to_timetable(vehicle_journey_idx, local_zone)?;
+        day_to_timetable.base.get(day, days_patterns).cloned()
+    }
+
+    pub fn get_realtime_timetable(
+        &self,
+        vehicle_journey_idx: &VehicleJourneyIdx,
+        local_zone: LocalZone,
+        day: DaysSinceDatasetStart,
+        days_patterns: &DaysPatterns,
+    ) -> Option<Timetable> {
+        let day_to_timetable = self.get_day_to_timetable(vehicle_journey_idx, local_zone)?;
+        day_to_timetable.real_time.get(day, days_patterns).cloned()
+    }
+
```

Comment:
Can we factor these two functions into one  ? 
```suggestion
   pub fn get_base_timetable(
        &self,
        vehicle_journey_idx: &VehicleJourneyIdx,
        local_zone: LocalZone,
        day: DaysSinceDatasetStart,
        days_patterns: &DaysPatterns,
        real_time_level : RealTimeLevel
    ) -> Option<Timetable>
```

Reply:
Done thank you